languages/python/sqlalchemy-oso/tests/test_roles2.py

```diff
@@ -150,7 +198,7 @@ def sample_data(init_oso, Organization, Repository, User, Issue):
 # - Passing a bad declarative_base to OsoRoles raises an exception
 
 
-def test_oso_roles_init(auth_sessionmaker, Base, User):
+def test_oso_roles_init(engine, auth_sessionmaker, Base, User):
```

Comment:
how was this passing before?

Reply:
lol I don't really know

Reply:
I think maybe because sqlalchemy will take a method that returns an engine as the engine so it was just using the fixture method directly to create an engine? Wild.